src/database.ts

```diff
@@ -92,8 +92,8 @@ export class Database<D = unknown> {
   //    D1-ORM is a good example: https://github.com/Interactions-as-a-Service/d1-orm/
   async batch<T = D>(
     statements: Statement[],
-    opts: Signal = {}
-    // reads returns an Array with legnth equal to the number of batched statements,
+    controller?: PollingController
```

Comment:
Couple things here:
1. This wasn't really an "options" object, it was a specific type of object. I usually think of "options" as an object that mixes different type of options, e.g., color, maxHeight, etc. I changed the param name to reflect.
2. The value of `opts` was `Signal`, but this method may call mutable methods that take polling options.